MergeRequestTable.elm

```diff
@@ -50,27 +50,23 @@ update action model =
             LT
         sortedList = sortWith mrComparison list
       in
-        ({ model | mrStats = sortedList }, Effects.tick Tick)
+        ({ model | mrStats = sortedList }, Cmd.none)
+
+    UpdateList (Err err) ->
```

Comment:
See above. I think this should be modeled as different message. Maybe `DisplayError`?